extensions/amp-sidebar/1.0/test/test-sidebar.js

```diff
@@ -57,12 +57,42 @@ describes.sandboxed('Sidebar preact component', {}, (env) => {
       openButton.getDOMNode().click();
       wrapper.update();
 
+      // verify sidebar is opened
+      let sidebarElement = wrapper.find(Sidebar).getDOMNode();
+      expect(sidebarElement).to.not.be.null;
+
+      // click on the backdrop
       const backdropElement = wrapper.find(Sidebar).getDOMNode().nextSibling;
       backdropElement.click();
       wrapper.update();
 
+      // verify sidebar closes
+      sidebarElement = wrapper.find(Sidebar).getDOMNode();
+      expect(sidebarElement).to.be.null;
+    });
+
+    it('should close the sidebar when the esc key is pressed', () => {
+      openButton.getDOMNode().click();
+      wrapper.update();
+
+      // verify sidebar is opened
+      let sidebarElement = wrapper.find(Sidebar).getDOMNode();
+      expect(sidebarElement).to.not.be.null;
+
+      // forces flush of effect queue (attaches esc key event listener)
+      wrapper.mount();
```

Comment:
This call causes the `useEffect` to be run

Reply:
cc: @rcebulko (this is the Preact side unit test).

amp-unit-test is above, it seems to work by waiting for the effect to run

Reply:
Nice! Thanks for tagging me!

Reply:
Thanks for the help in figuring this out!